### DISCUSSION TITLE
CRUD组件的from里面添加select 怎么动态绑定数据

### DISCUSSION THREAD

**Discussion by lovesL** [tolking/element-pro-components#487] [Q&A] [answered]
![image](https://github.com/tolking/element-pro-components/assets/27960630/80c2a7e9-35b2-4cee-af33-b49e8db6dfa9)
这样直接添加 select没有选项
declare 我在onMount调接口拿到的数据

**Comment by tolking** [comment, marked as answer]
异步数据需要使用 computed 包裹 formColumns

**Reply by lovesL** [reply]
怎么包啊 
![image](https://github.com/tolking/element-pro-components/assets/27960630/d22219a1-5b05-4e37-9fc8-fe94a0037d38)
这样嘛

**Comment by lovesL** [comment]
已解决
